src/components/signin/Mnemonics.js

```diff
@@ -34,7 +34,7 @@ const Mnemonics = ({ screenProps, navigation, styles }) => {
   const [showDialog] = useDialog()
   const [errorMessage, setErrorMessage] = useState()
   const [showErrorDialog, hideDialog] = useErrorDialog()
-  const input = React.createRef()
+  const input = useRef()
```

Comment:
i checked with global search - there's also a 4-5 places where createRef is used in the functional components.
Could you please find & fix them ?